v03_pipeline/lib/tasks/base/base_variant_annotations_table.py

```diff
@@ -16,20 +20,8 @@
 
 class BaseVariantAnnotationsTableTask(BaseUpdateTask):
     @property
-    def rdc_annotation_dependencies(self) -> dict[str, hl.Table]:
-        annotation_dependencies = {}
-        for rdc in ReferenceDatasetCollection.for_reference_genome_dataset_type(
-            self.reference_genome,
-            self.dataset_type,
-        ):
-            annotation_dependencies[f'{rdc.value}_ht'] = hl.read_table(
-                valid_reference_dataset_collection_path(
-                    self.reference_genome,
-                    self.dataset_type,
-                    rdc,
-                ),
-            )
-        return annotation_dependencies
+    def annotation_dependencies(self) -> dict[str, hl.Table]:
```

Comment:
Yeah, I actually needed to do it again and pushed a change right after you reviewed. `UpdateVariantAnnotationsTableWithUpdatedReferenceDataset` has rdc tables as dependencies but doesn't need the lookup table or gencode mapping. 

I put `other_annotation_dependencies` back into `UpdateVariantAnnotationsTableWithNewSamplesTask` where it has been since the recent change.